frontend/src/components/Logo/Logo.tsx

```diff
@@ -10,24 +10,12 @@ export const Logo = ({ isWithoutLogoTyp, height }: LogoProps) => {
   const logoHeight = height ? height : 48;
 
   if (isWithoutLogoTyp === 'small') {
```

Comment:
Mysle, ze tutaj tez mozna by po prostu zastosowac boolean np. hasImage? wtedy nie trzeba by bylo porownywac do stringa 'small' :)

Reply:
Też się nad tym zastanawiałam, ale rozmawiając z Piotrem to chyba mu chodziło o to żeby mieć wybór czy ma być logo z tekstem albo bez :)